src/sdk/pynni/nni/assessor.py

```diff
@@ -46,21 +45,20 @@ def trial_end(self, trial_job_id, success):
         trial_job_id: identifier of the trial (str).
         success: True if the trial successfully completed; False if failed or terminated.
         """
-        pass
```

Comment:
@leckie-chn Is this recommended style?
This will cause `W0107: Unnecessary pass statement (unnecessary-pass)`

Reply:
Currently in our `pylintrc` file `W1017` is not enabled. Are you sure you are using the right version of `pylintrc` file?

Reply:
I'm using the rc file included in this PR.

Reply:
You should use https://github.com/microsoft/nni/blob/master/pylintrc